packages/@aws-cdk/aws-appmesh/lib/gateway-route-spec.ts

```diff
@@ -44,10 +234,27 @@ export interface HttpGatewayRouteSpecOptions {
    * The VirtualService this GatewayRoute directs traffic to
    */
   readonly routeTarget: IVirtualService;
+
+  /**
+   * The default target host name to write to.
+   * Boolean is used to select either true = enable or false = disable.
+   * When enabled, rewrites the original request received at the Virtual Gateway to the destination Virtual Service name.
+   * When disabled, retains the original hostname/prefix from the request.
```

Comment:
Let's just kill the first two lines of the doc, I don't understand what they're saying.

```suggestion
   * When `true`, rewrites the original request received at the Virtual Gateway to the destination Virtual Service name.
   * When `false`, retains the original hostname/prefix from the request.
```

Reply:
Removed first two lines.